src/utils/formatProfileDataIntoArray.js

```diff
@@ -0,0 +1,26 @@
+import defaultIcon from "@/pesayetu/assets/icons/Group 4658-white.svg";
+
+export default function formatData(data) {
```

Comment:
This can move into `components/HURUmap/Panel` since it shouldn't be used anywhere else.